src/main/model/URLCommand.java

```diff
@@ -0,0 +1,76 @@
+package main.model;
+//not sure why we have this URL when there is one above it in the Curl class
+public class URLCommand {
+    private String URLs;
+    //not sure why we have this URL when there is one above it in the Curl class
+    private String company;
+    // not sure how this one fits in, but adding it anyhow
+    private String transactionTypes;
```

Comment:
You have good questions -- find answers! ;) What does this describe? What are you creating?